### PR TITLE
Add documentation checking, building and publishing workflow

### PR DESCRIPTION
**Pull request by pseusys**

### CODE REVIEW

**Comment by pseusys** [comment]
For future contributors.
In order to get documentation builder working you should:
1. Remove `docs/source/index.rst` line 19: `..`
2. Uncomment `makefile` line 69: `# sphinx-apidoc -e -f -o docs/source/apiref dff`
When documentation will be complete, `docs/source/comf.py` line 29 should look like 27 and 100 like 102 (see TODOs).
Good luck with documenting!

File: docs/source/conf.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # This file only contains a selection of the most common options. For a full
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
+import glob
```
Comment (kudep) [inline docs/source/conf.py:6]: glob is unused

**Review by kudep — CHANGES_REQUESTED** [review]
fixes:
- [ ] clear
- [ ] version
- [ ] glob

File: docs/source/conf.py (hunk 1 above)
Comment (RLKRo) [reply docs/source/conf.py:6]: It will be used when the line 100 is uncommented. I don't think we should remove the import just to add it back later.

Comment (kudep) [reply docs/source/conf.py:6]: I recommend just  remove it now and revert when it will be needed

**Comment by kudep** [comment]
pls, do not use force push like I do some time

**Review by kudep — APPROVED** [review]
Windows test do not passed because of psycopg2 was updated. Wait fixing of windows test in another PR

**Pull request merged**